buildSoup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -eu
+# Generate Soups
+
+DEPLOY_DOC=${DEPLOY_DOC:-soup}
+TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG:-owner/repo}
+APP="${TRAVIS_REPO_SLUG#*/}"
+GO111MODULE=on
+
+if [ -z "${TRAVIS_TAG+x}" ]; then
```

Comment:
typo?

Reply:
you mean the +x ?
it's a way of testing that TRAVIS_TAG is not set

Reply:
ok then